DCOS/autoscale-testing/AutoScale.Tests.ps1

```diff
@@ -67,6 +67,39 @@ function AreAllNodesHealthy ($RG_NAME) {
     return $isHealthy
 }
 
+function AllMetricsGood ($RG_NAME) {
+    $masterFQDN = getMasterFQDN($RG_NAME)
+    $res = Invoke-WebRequest "http://$masterFQDN/dcos-history-service/history/last" | ConvertFrom-Json | Select-Object slaves
+    $agentCount = $res.slaves.Count
```

Comment:
Where is this `$agentCount` variable used ?

Reply:
It was used only for debugging, it's no longer needed because if matching write-output was removed. I will remove it.